apps/src/lib/cli/utils.rs

```diff
@@ -257,7 +257,9 @@ where
     <T as FromStr>::Err: Debug,
 {
     pub fn def(&self) -> ClapArg {
-        ClapArg::new(self.name).long(self.name).multiple(true)
```

Comment:
`Deprecated since 3.0.0: Split into Arg::multiple_occurrences (most likely what you want) and Arg::multiple_values`